site/content/entry/citra-progress-report-2023-q1/index.md

```diff
@@ -0,0 +1,413 @@
+
+{{< figure src="binding.png"
+    title="Automatically bind your controls easily." >}}
+
+### Disable HIDAPI drivers due to compatibility problems with certain controllers ([#5123](https://github.com/citra-emu/citra/pull/5123), [#5179](https://github.com/citra-emu/citra/pull/5179)) by [vitor-k](https://github.com/vitor-k)
+
+The change of SDL2 from 2.0.8 to 2.0.10 broke some controllers that reported themselves as a Switch or Xbox One controller, such as DS4 controllers. This PR disables the HIDAPI drivers to allow for those affected controllers to begin to work again. HIDAPI drivers were enabled again for SDL2.0.12 and up in a [later PR](https://github.com/citra-emu/citra/pull/5179).
+
+### Implement official GameCube adapter support ([#5735](https://github.com/citra-emu/citra/pull/5735)) by [epicboy](https://github.com/ameerj)
+
+Ported over from yuzu, with code and feature improvements thanks to [german77](https://github.com/german77) and [lioncash](https://github.com/lioncash), this adds a whole new input device to Citra for you to use! Using the official GameCube adapter from Nintendo, you can now use the classic GameCube controllers to play your favorite games on Citra!
+
+Instructions on how to set this up can be found on the [yuzu FAQ](https://yuzu-emu.org/wiki/faq/#how-do-i-use-my-gamecube-controller-adapter).
+
+### Add support for SDL controller accelerometer/gyro events ([#5851](https://github.com/citra-emu/citra/pull/5851)) by [flibitijibibo](https://github.com/flibitijibibo)
+
+Following the release of SDL 2.0.14 came support for motion controls, something which had been anticipated for many years in SDL. In Citra, this can be used as an alternative to mouse controls or CemuhookUDP. This is a big benefit to many users, as setting up Cemuhook can be tedious and outright confusing at times. 
+Setting up SDL motion controls is as easy as pressing a button! (No, seriously. That is all you have to do.)
+
+{{< figure src="motion.png"
+    title="Motion controls at the press of a button!" >}}
+
+# Android
+
+If you missed the last announcement, this is a follow up to our previous [Citra Android Update](https://citra-emu.org/entry/citra-android-update/)!
+
+We are still updating Citra Android, but all builds are now hosted on our [Nightly repository](https://github.com/citra-emu/citra-nightly/releases) instead of the Google Play Store at the moment. This is due to Google's new storage permissions system, which requires a large rewrite of our file handling for Citra Android. We are actively working on getting updates for Citra Android back on the Google Play Store. 
```

Comment:
Please remove the mention of the Play Store as I intend to update this imminently.

Reply:
Also SAF PR is merged :) ... maybe just mention the Play Store is a bit outdated because of SAF, but that should change soon (?)

Reply:
> Please remove the mention of the Play Store as I intend to update this imminently.

Please wait until the 2nd part of the Vulkan PR is merged, since that includes Mali OpenGL optimizations (so it runs at more than 10 fps). Would suck to have people updating to the latest builds, checking things out after years only to find that the Playstore build is still unusable for Mali GPU users.

Reply:
Tend to agree, but we have a hard deadline of getting the Play Store app updated within about 5 weeks. I'd love to see Vulkan merged, but the Play Store build is getting imminently updated either way -- otherwise the app will get delisted due to lack of SAF. Therefore, I think it does not make sense to deliberately point out that users steer clear of it, we can just omit that.

Reply:
Is what I've put now for it good? It doesn't say to not use the Google Play Store, but also tells users that all of the new features listed aren't on it and to use the Nightly builds to use the features instead.